### ISSUE TITLE
Costs: Fertilizers - Missing option to select type of fertilizer - either "Organic Mulch" or "Applied NPK". The NPK applications only drop down with the selection of "Applied NPK"

### ISSUE THREAD

**Issue by pjconrad** [edanweis/LCC#98]

**Comment by CNimmo16** [comment]
Should this be implemented as two separate subcategories, one for "Organic mulch fertilizer" and one for "Applied NPK fertilizer", allowing you to enter quantities of both?

**Comment by pjconrad** [comment]
Yes, This should be two separate sub-categories, in theory you could use some of both.

**Comment by CNimmo16** [comment]
Unfortunately wasn't able to get this done, currently in the google sheets only the applied npk is being stored, I don't have access to firebase's cloud functions to change this. Fertilizers in general need to be fixed in the google sheets as currently they aren't properly integrated - co2 values also not calculating.

**Comment by edanweis** [comment]
@CNimmo16 I add you as Editor to the Firebase account using your gmail, from there you should be able to access cloud functions. Although I think it's just a matter of adding/checking that the Fertilizer inputs are in `parsing.xlsx` and imported as JSON into Firebase metadata with the correct cell references, and using the same component to render the slider input.

**Comment by CNimmo16** [comment]
@edanweis if you look at a user spreadsheet, eg https://docs.google.com/spreadsheets/d/1SHMP1BFXae5CixiLD3RcqfU4Dgc5zycEgx_Jnvg7IyA/edit#gid=1413427001, the fertilizer column which had a dropdown header in the excel sheet has been replaced by just a static column. Realistically I don't think I have time to fix this unfortunately.

**Comment by CNimmo16** [comment]
@pjconrad at the moment fertilizer co2 values are being calculated based on the total of planting area 1 2 and 3 in the attached screenshot (from "Carbon Sink - Shrubs"), multiplied by the relevant kg per application and frequency. As far as I can see nothing in the application is currently updating those planting area values, so for now the fertilizer co2 emissions all calculate to 0. I can change the spreadsheet to use the total area of moderate and intensive management lawn instead (the two lawn types that require fertilizer)?

![Screenshot from 2019-09-18 19-06-54](https://user-images.githubusercontent.com/48329714/65174287-85df1180-da48-11e9-83d5-44a840043de0.png)
